.github/workflows/ts_check.yml

```
@@ -22,7 +22,7 @@ jobs:
             - name: 🔧 Install the rust toolchain
               uses: dtolnay/rust-toolchain@stable
               with:
-                  toolchain: 1.91.0
+                  toolchain: 1.92.0
```

```suggestion
                  toolchain: 1.92
```
This way it would use the newest patch version, and not `1.92.0` specifically